storage/range.go

```diff
@@ -2028,6 +2043,7 @@ func (r *Range) AdminMerge(args *proto.AdminMergeRequest, reply *proto.AdminMerg
 					UpdatedDesc:    updatedDesc,
 					SubsumedRaftID: subsumedDesc.RaftID,
 				},
+				Intents: []proto.Key{desc1Key, desc2Key},
```

Comment:
What happens if desc1Key and desc2Key are on different ranges? (This could happen in a merge if a very large 'table' was deleted and all of its ranges were getting collapsed away to nothing; I'm not sure if it could happen in a split)